docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -447,151 +394,147 @@ where t is the period for which transition the is estimated (a year) eg. 365 day
 
    Note also that we used the same recovery duration for 0-6 months as 6-59 months in model 2. We have now updated the recovery durations for effectively treated population to be specific for 0-6 months in this model 4. 
 
-
-.. list-table:: Modeled Outcomes **Table 1** SAM 0-6 months
-  :widths: 5 5 20 15 15 15 15
+.. list-table:: Annual recovery rate equations
   :header-rows: 1
 
-  * - Outcome
-    - Outcome type
-    - Outcome definition
-    - Effect size
-    - Effect size application
-    - Value, source
+  * - Parameter
+    - Value
     - Note
-  * - :math:`r2_{ux}`
-    - transition probability
-    - Daily recovery probability of untx SAM
-    - :math:`\frac{t}{\text{median time-to-recovery (days) of utx SAM}}`
-    - See age, sex specific derivation below
-    - [Isanaka_2021]_ 
-    - Derived from Isanaka 2021 and tx time to recovery for >6mo
-  * - :math:`t1_{sam}`
-    - transition probability
-    - Daily recovery probability of tx SAM 
-    - :math:`\frac{t}{\text{median time-to-recovery (days) of tx SAM}}`
-    - time-to-recovery (mean) 13.3 (SD 6.9) days 
-    - [Vygen_2013]_
-    - Recovery time for 0-6m in SC in Niger
-
-.. list-table:: Modeled Outcomes **Table 2** SAM 6-59 months
-  :widths: 5 5 20 15 15 15 15
+  * - :math:`r_{SAM,ux}`
+    - :math:`\frac{k - r_{SAM,tx} * C * E_{SAM} - mortality_{SAM|a,s,l,y}}{1 - C * E_{SAM}}`
+    - See constant values in table below and equation derivation below table. Ali to confirm and potentially update this equation. Previously referred to as :math:`r2_{ux}`
+  * - :math:`r_{SAM,tx}`
+    - :math:`365 / (duration_\text{treated SAM}`
+    - See constant values in table below. Previously referred to as :math:`t1_{SAM}`
+  * - :math:`r_{MAM,ux}`
+    - :math:`365 / duration_\text{untreated MAM}`
+    - See constant values in table below. Previously referred to as :math:`r2_{ux}`
+  * - :math:`r_{MAM,tx}`
+    - :math:`365 / duration_\text{treated MAM}`
+    - See constant values in table below. Previously referred to as :math:`t2_{MAM}`
+
+.. warning::
+
+  The equation for the :math:`r_{SAM,ux}` defined in the table above along with the parameter values defined below results in negative values for :math:`r_{SAM,ux}` in the 1 - 6 month age group (also the neonatal age groups, but we are not modeling treatment in these groups, so we can ignore these). This is likely due to the higher mortality rate in the younger age groups and the not age-specific wasting durations used in these calculations. The research teams needs to update these equations/values accordingly so that we have plausible numbers.
```

Comment:
We are *temporarily* not modelling 0-6 month old wasting transitions. 
Also, this is likely due to the age-specific treated sam duration, not mortality.